components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2096,6 +2096,26 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         });
     }
 
+    async getNotifications(ctx: TraceContext): Promise<string[]> {
+        const result = await super.getNotifications(ctx);
+        const user = this.checkAndBlockUser("getNotifications");
+        if (user.usageAttributionId) {
```

Comment:
This has to be replaced by the `getBillingMode(user)` method. But as it's not there it, this is fine.